MediaBrowser.Model/Plugins/PluginInfo.cs

```diff
@@ -42,9 +57,13 @@ public class PluginInfo
         public bool CanUninstall { get; set; }
 
         /// <summary>
-        /// Gets or sets the image URL.
+        /// Gets or sets a value indicating whether this plugin has a valid image.
+        /// </summary>
+        public bool HasImage { get; set; }
```

Comment:
What is this used for?

Reply:
It permits this functionality #4583, and enables the use of the {Id}/Images API only as required.

Is set to true, if the manifest has defined an image and that image file is present within the plugin folder.

The system config option DisablePluginImages overrides and keeps things as they are.